tests/foreman/api/test_smartproxy.py

```diff
@@ -41,9 +42,10 @@ def setUpClass(cls):
         existing smart proxy should always succeed.
         """
         super(SmartProxyMissingAttrTestCase, cls).setUpClass()
-        smart_proxies = entities.SmartProxy().search()
-        assert len(smart_proxies) > 0
```

Comment:
+1 to what @abalakh said. Perhaps, just to make the assertion failure more descriptive, we could add a failure message, such as "No Capsules retrieved via API"